client/src/app.scss

```
@@ -355,7 +355,7 @@ sup.new {
 
 .sticky-header-container {
   position: sticky;
-  top: 0;
+  top: -0.1px;
```

```suggestion
  top: -1px;
```
The `-0.1px` doesn't work on Firefox developer 131.0b9 (64-bit) at 300% zoom:

https://github.com/user-attachments/assets/b10b4b09-509e-47bf-b221-7933ad53f5c4